include/openssl/pkcs7.h.in

```diff
@@ -38,12 +38,12 @@ extern "C" {
 #endif
 
 
-/*-
```

Comment:
I thought the changes to comments marked with `/*-` were going to be reverted?

Reply:
Again, as above, why mark this as resolved when it isn't? If I've missed a comment that says "X said it's okay to leave this in" then do me the courtesy of pointing me at it when marking this as resolved.